### PR TITLE
14.1.1 release

### PR DESCRIPTION
**Pull request by paurosello**
Related to https://github.com/giantswarm/giantswarm/issues/16585

### CODE REVIEW

File: aws/v14.1.1/release.diff [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    name: cert-manager                                                 name: cert-manager
+    version: 2.4.3                                                     version: 2.4.3
+  - name: chart-operator                                             - name: chart-operator
+    version: 2.13.0                                                    version: 2.13.0
+  - name: cluster-autoscaler                                         - name: cluster-autoscaler
+    version: 1.19.2                                                    version: 1.19.2
+  - componentVersion: 1.6.5                                          - componentVersion: 1.6.5
+    name: coredns                                                      name: coredns
+    version: 1.4.1                                                     version: 1.4.1
+  - componentVersion: 0.7.6                                     |    - name: external-dns
```
Comment (alex-dabija) [inline aws/v14.1.1/release.diff:24]: Component version is missing. It's probably because of devctl discarding it.

Comment (paurosello) [reply aws/v14.1.1/release.diff]: fixed

**Review by njuettner — APPROVED** [review]
lgtm

**Comment by paurosello** [comment]
Currently testing with AWSCNFM, will merge once its successful

**Review by alex-dabija — APPROVED** [review]

**Pull request merged**